src/VarDumper.php

```diff
@@ -39,6 +39,9 @@
  */
 final class VarDumper
 {
+    public const OBJECT_ID_PROPERTY = '__id__';
+    public const OBJECT_CLASS_PROPERTY = '__class__';
+    public const DEPTH_LIMIT_EXCEEDED_PROPERTY = '__depth_limit_exceeded__';
```

Comment:
A potential conflict is possible. Suggest use symobls that deny to use in object property name.